tests/unit/Core/Content/Product/Stock/OrderStockSubscriberTest.php

```diff
@@ -92,10 +100,17 @@ public function testBeforeWriteOnlyReactsToLiveVersions(): void
         $stockStorage = $this->createMock(StockStorage::class);
         $stockStorage->expects(static::never())->method('alter');
 
+        $systemConfigService = $this->createMock(SystemConfigService::class);
```

Comment:
We have a class StaticSystemConfigService, you put the needed configuration in the constructor, it's much nicer to handle than mocks